observability/README.adoc

```diff
@@ -19,18 +19,88 @@ workspace. Any modifications in your project will automatically take effect in t
 TIP: Please refer to the Development mode section of
 https://camel.apache.org/camel-quarkus/latest/first-steps.html#_development_mode[Camel Quarkus User guide] for more details.
 
+=== How to enable metrics
+To enable observability features in Camel, we need to add some additional dependencies to the project's pom.xml file.
```

Comment:
Also, here, use Camel Quarkus. I think it's better to avoid confusion (it's a bit different to enable that on Camel Core).